test/aspect_test_helper.rb

```diff
@@ -0,0 +1,8 @@
+Spork.prefork do
+  unless ActiveRecord::Base.connection.table_exists?(:test_aspects)
+    ActiveRecord::Base.connection.create_table :test_aspects do |t|
```

Comment:
Is it necessary to have a table to register an aspect (that's what the classes that use this helper do, right?)? How about mocking all db calls if all the table contains is the `host_id`?